cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitorTest.java

```diff
@@ -230,8 +230,8 @@ public void testBasicClusterModel() throws NotEnoughValidWindowsException {
                                                          new ModelCompletenessRequirements(2, 1.0, false),
                                                          new OperationProgress());
     assertEquals(6.5, clusterModel.partition(T0P0).leader().load().expectedUtilizationFor(Resource.CPU), 0.0);
-    assertEquals(6.5, clusterModel.partition(T0P0).leader().load().expectedUtilizationFor(Resource.NW_IN), 0.0);
-    assertEquals(6.5, clusterModel.partition(T0P0).leader().load().expectedUtilizationFor(Resource.NW_OUT), 0.0);
+    assertEquals(13, clusterModel.partition(T0P0).leader().load().expectedUtilizationFor(Resource.NW_IN), 0.0);
```

Comment:
I am not sure why we changed the test values in this class.

Reply:
Because now we have two metrics in NW_IN and NW_OUT. Each of them contributes 6.5, therefore the value needs to be changed, more specifically multiplied by the number of metrics in the group.